lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb

```diff
@@ -2946,8 +2950,13 @@ def airconditioner_variablerefrigerantflow_cooling_apply_cop(airconditioner_vari
     end
 
     # Set COP
-    airconditioner_variablerefrigerantflow.setRatedCoolingCOP(cop.to_f) unless cop.nil?
-
+    unless cop.nil?
+      if model.version < OpenStudio::VersionString.new('2.9.0')
```

Comment:
You need to get `model` first from the AC_VRF object

Reply:
should've checked that...